string robot_name
string joint_name
---
string result
